2-mandatory/3-stocks.js

```diff
@@ -64,11 +86,31 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
```

Comment:
This looks good to me.
Can you think about ways to make this a little easier to read?
One suggestion might be to have a separate function to calculate the highest price in an array of prices. Then we can call that new function from this one.